src/renderer/entities/governance/ui/LockValueDiff.tsx

```diff
@@ -15,7 +15,8 @@ export const LockValueDiff = ({ from, to, asset }: Props) => {
   const fromBN = BN.isBN(from) ? from : new BN(from);
   const toBN = BN.isBN(to) ? to : new BN(to);
 
-  const diff = fromBN.sub(toBN).isNeg() ? fromBN.sub(toBN).abs() : BN_ZERO;
+  const realDiff = fromBN.sub(toBN);
+  const diff = realDiff.isNeg() ? realDiff.abs() : realDiff;
```

Comment:
The point is: if the current lock is less than the locks you have already - there will be no difference because locks will be reused from the current lock amount

Reply:
Oh, okay, i'll revert that

Reply:
Added small description about behavior